src/main/java/data/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package data.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruitsStorage = new HashMap<>();
```

Comment:
According to the encapsulation principle, the `fruitsStorage` map should have a private access modifier to hide its inner structure and prevent direct modification from outside of the `Storage` class. Consider providing public methods to interact with the storage instead.